tools/data_renewal_generate.py

```diff
@@ -0,0 +1,149 @@
+import os
```

Comment:
Specify the env to run with py3.

```suggestion
#!/usr/bin/env python3

import os
```

Reply:
Thanks, good catch.  I'll add it.